### ISSUE TITLE
static:warm command takes very long to execute

### ISSUE THREAD

**Issue by m-glass** [statamic/cms#4730]
### Bug description

When running `php please static:warm`, the routes are getting called sequentially. When executing on a very large page (content-heavy), the command takes very, very long to execute (40 min or longer). Maybe making it executing concurrently would solve this issue.

### How to reproduce

1. Be very enthusiastic about your content and write much content
2. Execute `php please static:warm` to wait for eternity

### Versions

Statamic 3.2.20 Pro
Laravel 8.46.0
PHP 7.4.25
transformstudios/review 3.1
withcandour/aardvark-seo 2.0.16

### Installation

Other (please explain)

### Additional details

Existing content-rich Statamic page

**Comment by arthurperton** [comment]
Hi @m-glass, thanks for your report. 

The requests are running concurrently. Since version 3.2.28 you can also configure the amount of requests that are sent simultaneously by setting the `warm_concurrency` for the strategy you are using in your `static_caching.php` config file. There is no magic number and performance will vary with the available resources of course.  It defaults to 25, but changing the number might help with speeding things up in your case.

I am going to close this issue for now, since there is configurable concurrency now. Please feel free to reopen or open a new one if you think there is still a bug.

**Comment by callum-veloxcommerce** [comment]
Having upped concurrency to 100, this has made virtually no difference to the performance of the command. If anything it actually seems slower now.

**Comment by ehbfa** [comment]
Hi @callum-veloxcommerce - you can add the `--queue` flag to the end of the command and then all the URLs will be added to the queue runner. As long as you do NOT have QUEUE_CONNECTION in your .env set to sync. 

This way you will probably remain under the 10 minute mark for Forge. The entries, taxonomies, taxonomy terms and custom routes will still be processed directly, so the script will still take up some time, but not nearly as long as without a queue flag. 

So to summarize: 

`php please static:warm --queue`
